### PR TITLE
Jarv/connect to prod clone

### PR DESCRIPTION
**Pull request by jarv**

### CODE REVIEW

File: playbooks/edx-east/connect_sandbox.yml [hunk 1]
```diff
@@ -0,0 +1,110 @@
...
+
+    - name: restart mongodb
+      service: >
+        name=mongodb
+        state=restarted
+      tags: update_mongo_data
+
+    - name: grab the most recent backup from s3 for forums
+      shell : >
+        /edx/bin/s3cmd ls s3://edx-mongohq/mongohq_backups/ | grep comment | sort | tail -1  | awk '{ print $4 }'
```
Comment (feanil) [inline playbooks/edx-east/connect_sandbox.yml:28]: Is s3cmd guaranteed to be installed?

File: playbooks/edx-east/connect_sandbox.yml [hunk 2]
```diff
@@ -0,0 +1,110 @@
...
+        - user: cs_comments_service
+          password: password
+          database: cs_comments_service
+        - user: exdapp
+          password: password
+          database: edxapp
+
+      # WARNING - calling lineinfile on a symlink
+      # will convert the symlink to a file!
+      # don't use /edx/etc/serv-ars.yml here
```
Comment (feanil) [inline playbooks/edx-east/connect_sandbox.yml]: typo `server-vars.yml`

**Comment by feanil** [comment]
@jarv other than the two comments, this looks good.  I think it would make sense to install s3cmd as part of this playbook since it relies on it.

File: playbooks/edx-east/connect_sandbox.yml (hunk 1 above)
Comment (jarv) [inline playbooks/edx-east/connect_sandbox.yml:28]: Comes with the aws role which can be assumed for sandboxes.
We install it for the s3 log rotation job.

**Comment by feanil** [comment]
:+1:  per conversations in person since we know that s3cmd will be available on the sandboxes.

**Pull request merged**